runtime/fs_fake.js

```diff
@@ -62,15 +63,64 @@ MlFakeDevice.prototype.exists = function(name) {
   this.lookup(name);
   return this.content[name]?1:0;
 }
-MlFakeDevice.prototype.mkdir = function(name,mode) {
+MlFakeDevice.prototype.mkdir = function(name,mode, raise_unix) {
+  var unix_error = raise_unix && caml_named_value('Unix.Unix_error');
+  if(this.exists(name)) {
+    if (unix_error) {
+      caml_raise_with_args(unix_error, [8, caml_string_of_jsstring("mkdir"), caml_string_of_jsstring(this.nm(name))]);
```

Comment:
We need a readable way to reference the error constructor.

Reply:
@hhugo I added another function in https://github.com/ocsigen/js_of_ocaml/pull/1115/commits/fad310827a9b0829ae60572f5281b1f11276c3ff called `make_unix_err_args` that allows the string named to be used. Do you think that is a good solution?